adapter-http/src/main/resources/activities/baselines/http-rest-keyvalue.yaml

```diff
@@ -19,9 +19,8 @@ bindings:
   #   multiple hosts: restapi_host=host1,host2,host3
   #   multiple weighted hosts: restapi_host=host1:3,host2:7
   weighted_hosts: WeightedStrings('<<restapi_host:stargate>>')
-  # http request id
   request_id: ToHashedUUID(); ToString();
-
+  request_token: ToString(); TextOfFile("data/stargate_token.txt");
```

Comment:
It would be nice to allow this file to change, as `TextOfFile("TEMPLATE(stargate_tokenfile,data/stargate_token.txt)")`